assets/js/index.js

```diff
@@ -80,12 +91,18 @@ function formatDate(date) {
 function showTasks(tasks) {
   todoList.innerHTML = tasks.sort(
     (a, b) => new Date(b.createdAt) -  new Date(a.createdAt)
-  ).map(({text, createdAt}) => `<li class="todo-item">
+  ).map(({ text, createdAt, id: taskId }) => `<li id="${taskId}" class="todo-item">
     <input type="checkbox" />
     <p class="todo"  title="${formatDate(createdAt)}">${text}</p>
     <i class="fa-solid fa-pen-to-square edite-icon"></i>
     <i class="fa-solid fa-xmark remove-icon"></i>
     </li>`).join("");
+  const removeIcons = Array.from(document.getElementsByClassName("remove-icon"));
+  removeIcons.forEach(removeIcon => {
+    removeIcon.addEventListener("click", function (event) {
+      deleteTodo(event.target.parentNode.id);
+    })
+  });
```

Comment:
Extract this to another function and call it `bindDeleteEvent`.

Reply:
Done.